packages/shorebird_cli/lib/src/commands/release_new/release_type.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
+
+/// The different types of shorebird releases that can be created.
+enum ReleaseType {
+  /// An Android archive used in a hybrid app.
+  aar,
```

Comment:
for consistency might be nice to rename these to `hybridAndroid` and `hybridIos` since it's a bit weird to have some names refer to the artifact types (aar, iosFramework) and other names refer to higher level concepts (android, ios)